.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,57 @@
+name: Bug report
+description: Help us improve InstantSearch
+labels:
+  - triage
+body:
+- type: textarea
+  attributes:
+    label: 🐛 Bug description
+    description: A clear and concise description of what the bug is.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: 🔍 Bug reproduction
+    description: Steps to reproduce the behavior.
+    placeholder: |
+      1. Go to ...
+      2. Click on ...
+      3. Scroll down to ...
+      4. See error
+
+      You can attach screenshots by dragging & dropping, selecting or pasting them on the textarea.
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Live reproduction
+    description: |
+      A live example helps a lot! Fork the sandbox relevant to your package, reproduce the bug and paste the URL here.
+      Sandboxes: [InstantSearch.js](https://codesandbox.io/s/github/algolia/create-instantsearch-app/tree/templates/instantsearch.js)
```

Comment:
This URL is outdated, CISA is now in the monorepo.

Reply:
Oh that's true, thanks for catching that!